source/includes/Visit/visits_configuration.md

```diff
@@ -0,0 +1,145 @@
+# Visits Configuration
+
+## Visits Configuration Overview
+
+Visits introduce set of settings related to Visits Behavior. You can retrieve and update this configuration through REST API.
```

Comment:
Can you correct the grammar above?

Reply:
I'm not sure where is the grammar error. Could you point it out?

Reply:
After `introduce`

Reply:
I meant set of settings, that is multiple settings :)

Reply:
a set of settings

Reply:
Resolved here: https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/151/commits/34e26613f5d59ae674dcc197a3c094cb2448c576